servers/quarkus-server/src/main/resources/application.properties

```diff
@@ -28,7 +28,7 @@ nessie.version.store.jgit.type=DISK
 ### where to put your git repository. Only used if nessie.version.store.jgit.type=JGIT
 nessie.version.store.jgit.directory=/tmp/jgit
 ### backing store for jgit dynamo version store. Either INMEMORY or DYNAMO. Typically DYNAMO is used unless performing a specific test
-nessie.backends.type=DYNAMO
```

Comment:
Why has this changed?

Reply:
I changed it to INMEMORY to get it working initially. I'll revert that.